#
# Copyright (c) 2010, 2022 Oracle and/or its affiliates. All rights reserved.
#
# This program and the accompanying materials are made available under the
# terms of the Eclipse Distribution License v. 1.0, which is available at
# http://www.eclipse.org/org/documents/edl-v10.php.
#
# SPDX-License-Identifier: BSD-3-Clause
#

WSS1500.invalid.usernameHandler=WSS1500 : le gestionnaire de nom utilisateur n'a pas \u00E9t\u00E9 configur\u00E9 correctement \u00E0 l'aide de la fonction callback et il est NULL (non configur\u00E9).

WSS1501.no.password.validator=WSS1501 : le valideur de mot de passe n'est pas sp\u00E9cifi\u00E9 dans la configuration

WSS1502.unsupported.digestAuth=WSS1502 : l'authentification Digest n'est pas prise en charge pour la demande de mot de passe

WSS1503.unsupported.requesttype=WSS1503 : type de demande non pris en charge pour la validation du mot de passe

WSS1504.unsupported.callbackType=WSS1504 : type de callback non pris en charge d\u00E9tect\u00E9

WSS1505.failedto.getkey=WSS1505 : exception d'E/S d\u00E9tect\u00E9e : impossible d'obtenir la cl\u00E9/le certificat \u00E0 partir du fichier de cl\u00E9s (il ne s'agit pas n\u00E9cessairement d'une exception d'E/S)

WSS1506.invalid.SAMLPolicy=WSS1506 : l'assertion SAML ne figure pas dans la strat\u00E9gie.

WSS1507.no.SAMLCallbackHandler=WSS1507 : aucun gestionnaire de callback SAML n'a \u00E9t\u00E9 sp\u00E9cifi\u00E9 dans la configuration alors que cela est obligatoire : impossible de remplir l'assertion SAML

WSS1508.failed.validateSAMLAssertion=WSS1508 : \u00E9chec de la validation de l'assertion SAML

WSS1509.failed.init.truststore=WSS1509 : \u00E9chec de l'initialisation du truststore

WSS1510.failed.init.keystore=WSS1510 : \u00E9chec de l'initialisation du fichier de cl\u00E9s

# PEER is an alias name for the certificate. Should not be translated.
WSS1511.failed.locate.peerCertificate=WSS1511 : erreur lors de la localisation du certificat d'entit\u00E9 PEER dans le truststore.

WSS1512.failed.locate.certificate.privatekey=WSS1512 : erreur lors de la localisation du certificat par d\u00E9faut et de la cl\u00E9 priv\u00E9e dans le fichier de cl\u00E9s.

WSS1513.exception.validate.timestamp=WSS1513 : une exception est survenue lors de la validation de l'horodatage : erreur lors de l'analyse de l'horodatage UTC au format de date.

WSS1514.error.aheadCurrentTime=WSS1514 : l'heure en cours est ult\u00E9rieure \u00E0 l'heure d'expiration dans l'horodatage

WSS1515.error.currentTime=WSS1515 : l'heure de cr\u00E9ation est ant\u00E9rieure \u00E0 la valeur currenttime moins timestamp-freshness-limit moins max-clock-skew

WSS1516.error.creationAheadCurrent.time=WSS1516 : l'heure de cr\u00E9ation est ult\u00E9rieure \u00E0 l'heure en cours

WSS1517.X509.expired=WSS1517 : le certificat X509Certificate a expir\u00E9.

WSS1518.failedto.validate.certificate=WSS1518 : \u00E9chec de la validation du certificat

WSS1519.no.digest.algorithm=WSS1519 : l'algorithme Digest SHA-1 est introuvable

WSS1520.error.getting.rawContent=WSS1520 : erreur lors de l'obtention du contenu Raw du certificat

WSS1521.error.getting.userClass=WSS1512 : impossible de localiser la classe d'utilisateur

WSS1522.error.getting.longValue=WSS1522 : erreur lors de l'obtention de la valeur Long

WSS1523.error.getting.newInstance.CallbackHandler=WSS1523 : erreur lors de l'obtention de la nouvelle instance du gestionnaire de callback

WSS1524.unableto.resolve.URI.WSIT_HOME.notset=WSS1524 : impossible de r\u00E9soudre l'URL de configuration indiqu\u00E9e dans le fichier WSDL car la propri\u00E9t\u00E9 syst\u00E8me WSIT_HOME n'est pas d\u00E9finie

WSS1525.invalid.passwordHandler=WSS1500 : le gestionnaire de mot de passe n'a pas \u00E9t\u00E9 configur\u00E9 correctement \u00E0 l'aide de la fonction callback et il est NULL (non configur\u00E9).

WSS1526.failedto.getcertificate=WSS1526 : une exception d'E/S est survenue : \u00E9chec de l'obtention du certificat \u00E0 partir du truststore

WSS1527.X509.notValid=WSS1527 : \u00E9chec de la validation du certificat X509Certificate.

WSS1528.failed.initialize.key.password=WSS1528 : erreur lors de l'obtention du mot de passe de la cl\u00E9 du fichier de cl\u00E9s.

WSS1529.exception.in.certstore.callback=WSS1529 : exception dans CertStoreCallback
WSS1530.exception.in.certstore.lookup=WSS1530 : exception lors de la recherche de l'\u00E9l\u00E9ment CertStore
WSS1531.exception.instantiating.certselector=WSS1531 : exception lors de l'instanciation de l'\u00E9l\u00E9ment CertSelector fourni par l'utilisateur
WSS1532.exception.instantiating.aliasselector=WSS1532 : exception lors de l'instanciation de l'\u00E9l\u00E9ment AliasSelector fourni par l'utilisateur
WSS1533.X509.SelfSignedCertificate.notValid=WSS1533 : \u00E9chec de la validation du certificat auto-sign\u00E9.
WSS1534.no.keystore.set.in.keystorecallback.callbackhandler=WSS1534 : aucun fichier de cl\u00E9s n'est d\u00E9fini dans l'\u00E9l\u00E9ment KeyStorCallback par CallbackHandler
WSS1535.error.keystore.using.callback=WSS1535 : exception lors de l'obtention du fichier de cl\u00E9s \u00E0 l'aide du callback de fichier de cl\u00E9s
WSS1536.no.truststore.set.in.truststorecallback=WSS1536 : aucun truststore n'est d\u00E9fini dans l'\u00E9l\u00E9ment TrustStorCallback par CallbackHandler
WSS1537.error.truststore.using.callback=WSS1537 : exception lors de l'obtention du truststore \u00E0 l'aide du callback de truststore
WSS1538.error.getting.privateKey=WSS1538 : exception lors de l'obtention de la cl\u00E9 priv\u00E9e
WSS1539.privateKey.null.error=WSS1539 : l'\u00E9l\u00E9ment PrivateKey renvoy\u00E9 par PrivateKeyCallback \u00E9tait NULL
WSS1540.keystore.not.found.check.config=WSS1540 : impossible de localiser le fichier de cl\u00E9s ; v\u00E9rifiez l'assertion de fichier de cl\u00E9s dans la configuration WSIT
